lstchain/scripts/onsite/onsite_create_cat_B_calibration_files_with_batch.py

```diff
@@ -117,16 +116,21 @@ def main():
         if filters_list is not None:
             filters = filters_list[i]
 
-        input_files = find_interleaved_subruns(run, r0_dir, dl1_dir)    
+        # look in R0 to find the date
+        r0_list = find_r0_subrun(run,0,r0_dir)
+        date = r0_list.parent.name
+
+        # find input path
+        ver=version.rsplit(".")
```

Comment:
mmh,  prod_id can be also some general name, which does not fit to the version format

Reply:
I agree that it could be any string not in the format "vW.X.Y.Z", although you foresee including the initial "v" in the prod_id variable. On the other hand, using directly the currently used lstchain version would force you to run lstchain with the same "major" lstchain version (which I think would be desirable). I do have not a strong opinion on this, just bear in mind the previous comments.

Reply:
mmh, indeed it is not so clear. The initial idea for the prod_id trailet was to perform tests on calibration (so the prod_id was mainly or the output tree and could have any name ), but actually at the end I did not really use it a lot. Perhaps the trailet could be eliminated